<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片处理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(234, 255, 228);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="img1_Lable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #deffe1;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="img2_label">
        <property name="styleSheet">
         <string notr="true">background-color: #deffe1;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 255, 209);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="chooseBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #4CAF50;  /* 绿色 - 正常状态 */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 12px 24px;
    font-size: 20px;
font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;  /* 深绿色 - 悬停 */
    transform: scale(1.02);     /* 轻微放大（需代码实现动画） */
}</string>
        </property>
        <property name="text">
         <string>选择图片</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #4CAF50;  /* 绿色 - 正常状态 */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 12px 24px;
    font-size: 20px;
font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;  /* 深绿色 - 悬停 */
    transform: scale(1.02);     /* 轻微放大（需代码实现动画） */
}</string>
        </property>
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="returnBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: #f44336;  /* 红色 - 正常 */
    color: white;
    border-radius: 4px;
    padding: 10px 18px;
	font-size: 20px;
	font-weight: bold;
    
}

QPushButton:hover { background: #e53935; }  /* 悬停 */
QPushButton:pressed { 
    background: #d32f2f; 
    padding-left: 20px;  /* 按下时左移2px拟真 */
}</string>
        </property>
        <property name="text">
         <string>返回主界面</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
